docs/push-v2.md

```diff
@@ -1,5 +1,55 @@
 # Push notifications as a Nextcloud client device
 
+## Introduction
+
+> Why is push-notifications.nextcloud.com necessary?
+
+The mobile apps people install from the Google Playstore and Apple App Store are signed with Nextcloud developer keys or certificates.
```

Comment:
```suggestion
The Talk mobile apps from the Google Playstore and Apple App Store are signed with Nextcloud developer keys or certificates.
```

Reply:
The push proxy is used for the files apps as well.

Reply:
Then maybe simply Nextcloud mobile apps?